HostLibraryTests/CMakeLists.txt

```diff
@@ -78,21 +78,7 @@ if(TENSILE_USE_HIP)
 endif()
 
 if(TENSILE_USE_OPENMP)
-    #set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_OPENMP")
-    find_package(OpenMP QUIET)
-    if (OPENMP_FOUND)
-        set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
-        set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-        set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
-    else()
-        if(EXISTS /etc/redhat-release)
-            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp=libgomp")
-        else()
-            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp")
-            set(OPENMP_LIBRARY /usr/lib/x86_64-linux-gnu/libomp.so)
-            set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OPENMP_LIBRARY}")
-        endif()
-    endif()
+    find_package(OpenMP REQUIRED)
```

Comment:
For this build environment in rocBLAS we dropped the REQUIRED from this line and it was able to find the header file in order to compile.   Runtime use of OpenMP would thus likely fail so you would only get 1 thread.  Not sure if this HostLibraryTests will suffer much under one thread in this non-standard build environment.

Reply:
I don't think the find_package(OpenMP,...) in HostLibraryTests/CMakeLists.txt was the cause of the rocBLAS build failure; the one in Tensile/Source/CMakeLists.txt was.  In Tensile, we need far more than the header file.  For CPU reference result computation, OpenMP library is needed.  Sure, I can remove the REQUIRED from both places and the rocBLAS build will succeed, but then standalone benchmarking breaks:
```
-- Found OpenMP_C: -fopenmp (found version "4.5")
-- Could NOT find OpenMP_CXX (missing: OpenMP_CXX_FLAGS OpenMP_CXX_LIB_NAMES)
-- Could NOT find OpenMP (missing: OpenMP_CXX_FOUND) (found version "4.5")
LLVMObjectYAML_LIBRARY: /usr/lib/llvm-6.0/lib/libLLVMObjectYAML.a
-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   program_options
--   filesystem
--   system
-- Found ROCmSMI: /opt/rocm/lib/librocm_smi64.so
-- Configuring done
CMake Error at client/CMakeLists.txt:39 (add_executable):
  Target "tensile_client" links to target "OpenMP::OpenMP_CXX" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
```